shared/admin-tools/user/admin.js

```diff
@@ -76,6 +76,55 @@ const deactivateAdminAccount = async () => {
     .promise();
 };
 
+/**
+ * This verifies that the USTC admin user is disabled in Cognito
+ */
+const verifyAdminUserDisabled = async ({ attempt }) => {
```

Comment:
Can this be tested?

Reply:
with an automated test or do you mean manually?

Reply:
here is successful use of it in CI under the step "Deploy - Verify USTC Admin User Disabled" https://app.circleci.com/pipelines/github/flexion/ef-cms/37028/workflows/f892c4fe-8807-4e8b-a748-9852a433e78f/jobs/238735

Reply:
I meant automated, but I realize it'd require mocking `cognito.getAdminUser()`

Reply:
Ah gotcha. Looking into it!

Reply:
Well, I'm able to mock `getAdminUser` but it's starting to get tricky once I get into `getUserPoolId` which is called inside that separate `util.js` file. That's instantiating its own `CognitoIdentityServiceProvider` then returning `UserPoolCache[ENV]`. Getting kind of stuck there, if you have any ideas let me know! Unfortunately we don't have much test setup in the admin-tools folder for some help scaffolding up something.

Reply:
Ooh maybe I just needed to write that out because I think I spoke too soon and might have more of a direction now.